tests/views/playingScreen-test.js

```diff
@@ -104,6 +106,9 @@ describe('PlayingScreen', function () {
 
     TestUtils.Simulate.mouseOver(screen);
     expect(over).toBe(true);
+
+    TestUtils.Simulate.keyUp(screen, {key: 'Tab', which: 9, keyCode: 9});
```

Comment:
There are some examples of spying on callbacks on the 'Controller testing Anamorphic videos fix' in controller-test.js